api/src/test/java/io/opentelemetry/metrics/LongCounterTest.java

```diff
@@ -86,6 +87,8 @@ public void noopBind_WithBadLabelSet() {
   public void doesNotThrow() {
```

Comment:
looks like we need a test that asserts that we can't increment with a negative value.

Reply:
Done.